arch/arm64/boot/dts/qcom/sm7150-xiaomi-tucana.dts

```diff
@@ -28,40 +40,54 @@
 
 	qcom,board-id = <42 0>;
 
+	vreg_touch_vdd: regulator-touch-vdd {
+		compatible = "regulator-fixed";
```

Comment:
Try to keep the order of parameters as in dt-bindings example:
https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/regulator/fixed-regulator.yaml#L125